dimod/discrete/discrete_quadratic_model.py

```diff
@@ -421,20 +421,25 @@ def get_quadratic_case(self, u, u_case, v, v_case):
 
     def add_constraint_as_quadratic(self, terms: Linear, lagrange_multiplier: float,
                                     constant: float):
-        """
-        Add a linear constraint of the form \\sum_{ik} a_{ik} x_{ik} + C = 0 to dqm object as a quadratic objective.
+        """Add a linear constraint of the form below
```

Comment:
How about
```
    """Add a linear constraint as a quadratic objective

    Adds a linear constraint of the form :math:`\sum_{i,k} a_{i,k} x_{i,k} + C = 0`
    to the discrete quadratic model as a quadratic objective.

    Args:
        ...
```

The reason is that was sphinx can pick it up for the summary. You'll also want to add this method to the docs for the DQM, see https://github.com/dwavesystems/dimod/blob/f4d2744e9514e6d0ab41fd9637bdfe021919029b/docs/reference/dqm.rst